Cli-CredentialHelper/Installer.cs

```diff
@@ -202,11 +202,11 @@ public void DeployConsole()
             {
                 System.Security.Principal.WindowsIdentity identity = System.Security.Principal.WindowsIdentity.GetCurrent();
                 System.Security.Principal.WindowsPrincipal principal = new System.Security.Principal.WindowsPrincipal(identity);
-                if (!principal.IsInRole(System.Security.Principal.WindowsBuiltInRole.Administrator))
-                {
-                    DeployElevated();
-                    return;
-                }
+                //if (!principal.IsInRole(System.Security.Principal.WindowsBuiltInRole.Administrator))
```

Comment:
Wait - this needs to use elevation to install - Git is usually under `%ProgramFiles%`, which requires elevated privileges to modify. Did the comment out slip in or is it intentional?